ledger/src/bank_forks.rs

```diff
@@ -68,6 +72,8 @@ impl BankForks {
             root: 0,
             snapshot_config: None,
             last_snapshot_slot: bank_slot,
+            last_purge_zla_slot: bank_slot,
```

Comment:
What's a `zla`?

Reply:
zero lamport account

Reply:
heh, that I would have never guessed!

Reply:
ha. yea might need a better name for that.